acceptance/steps/manifest.go

```diff
@@ -33,3 +35,12 @@ func iApplyKubernetesManifest(ctx context.Context, t framework.TestingT, manifes
 
 	t.ApplyManifest(ctx, file.Name())
 }
+
+func iKustomizeApply(ctx context.Context, t framework.TestingT, url string) {
```

Comment:
This helper is net new to this commit. v25.2+ doesn't need to worry about it and the existing strategy of using the in repo CRDs felt kludgey.